bpf-loader/src/btf_helpers.c

```diff
@@ -0,0 +1,95 @@
+/* SPDX-License-Identifier: MIT
+ *
+ * Copyright (c) 2023, eunomia-bpf
+ * All rights reserved.
+ */
+
+#include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/utsname.h>
+#include <zlib.h>
+#include "btf_helpers.h"
+
+#define FIELD_LEN 65
+#define ID_FMT "ID=%64s"
+#define VERSION_FMT "VERSION_ID=\"%64s"
```

Comment:
There is a limit on the length of `sscanf()` here, and the hint of codefactor is unnecessary.

Reply:
OK，just ignore it.